core/src/main/java/hudson/FilePath.java

```diff
@@ -124,9 +125,8 @@
 import org.kohsuke.stapler.Stapler;
 
 import static hudson.FilePath.TarCompression.GZIP;
-import static hudson.Util.deleteFile;
-import static hudson.Util.fixEmpty;
-import static hudson.Util.isSymlink;
+import static hudson.Util.*;
```

Comment:
I would recommend to avoid doing it in a production code to avoid ambiguity

Reply:
I thought I had set Intellij to avoid doing this! I have fixed the settings & the code